pkg/model/transformer_test.go

```diff
@@ -18,6 +20,9 @@ func (c *fixedClock) Now() time.Time {
 }
 
 func TestTransformer_Transform(t *testing.T) {
+	err := os.Setenv("TRIVY_VERSION", "0.1.6")
```

Comment:
This will cause side effects for any following tests that get run after this as it's not cleared/cleaned up.